__std__/nix/toolchain/packages/check-the-flake.nix

```diff
@@ -1,29 +1,41 @@
 { inputs, cell }:
 
+# TODO(std) path must be fixed once __std__ is brought to the toplevel.
+# TODO(std) make this part of CI
```

Comment:
Either way, it may be considered temporary. However, maybe not ultra short term.

I may have mentioned this already: I'm planning to discover build targets from the ~~organelle clades~~ Cell Block Types.

But this is just yet in the design phase. So I just want to add this as ancillary information.

Reply:
Ok. I'm just leaving it like this for now as I'm using it a lot. 
I added a TODO to turn it into a `recurseForDerivations` right after stdization is completed.